rossum_api/clients/external_async_client.py

```diff
@@ -558,7 +558,7 @@ async def retrieve_organization(self, org_id: int) -> OrganizationType:
         organization = await self._http_client.fetch_one(Resource.Organization, org_id)
         return self._deserializer(Resource.Organization, organization)
 
-    async def retrieve_own_organization(self) -> OrganizationType:
+    async def retrieve_my_organization(self) -> OrganizationType:
```

Comment:
I actually prefer `retrieve_own_organization`, we even use that in our code :wink: